lib/task/propel/propelBuildNestedSetTask.class.php

```diff
@@ -65,10 +66,26 @@ public function execute($arguments = array(), $options = array())
       'menu' => 'QubitMenu'
     );
 
+    $excludeTables = array();
+
+    if (!empty($options['exclude-tables']))
+    {
+      $excludeTables = explode(',', $options['exclude-tables']);
```

Comment:
Suggestion: It might be worth wrapping this in a `trim()` to remove leading or trailing whitespace in the list.

Reply:
Oh, it's an array - so I guess it would have to use `array_map()` or similar to remove white space.

Reply:
Good call @djjuhasz, thanks!